docs/dependent_images.md

```diff
@@ -1,23 +1,77 @@
 ---
-title: Building on a Base Image
-description: "With multiple services in your app, a common pattern is to create a common base image with common dependencies"
+title: Getting Started with Image Builds
+description: "How to go from building one image to building multiple images"
 layout: docs
 sidebar: guides
 ---
 
+This is a brief intro to automatically build and deploy images
+to a local dev environment.
+
+We'll start with a single image.
+
+## Going from `docker build` to `docker_build`
+
+On the command-line, an image build looks like this:
+
+```shell
+docker build -t my-image .
+```
+
+`my-image` is the image name. `.` is the directory of files that you're including in the image.
+
+To make this part of your dev environment, add this to your Tiltfile:
+
+```python
+docker_build('my-image', '.')
+```
+
+`.` is still the directory of files you're including. Tilt will automatically watch them for changes.
+
+`my-image` is an image selector. Tilt will scan all your deployment YAML for
```

Comment:
Suggest talking about manifests instead of YAML to cover situations where YAML wasn't used
```suggestion
`my-image` is an image selector. Tilt will scan all your workload manifests for
```